### PR TITLE
Update README.md to add dependency Telescope to install instructions

### PR DESCRIPTION
**Pull request by GotaLoveFiraCode**
The title says it all.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,9 @@ return {
       '<CMD>lua require("activate").list_plugins()<CR>',
       desc = "Plugins",
     },
+  },
+  dependencies = {
+    { 'nvim-telescope/telescope.nvim', branch = '0.1.x', dependencies = { 'nvim-lua/plenary.nvim' } }
```
Comment (roobert) [inline README.md:75]: This doesn't quite look correct, I think the second dependencies key is unnecessary?

Is specifying the branch also necessary?

Comment (GotaLoveFiraCode) [reply README.md:75]: For telescope 'stable' yeah. And telescope needs plenary, so you need to add it as a dependency.

From the [telescope github](https://github.com/nvim-telescope/telescope.nvim):
```lua
{
    'nvim-telescope/telescope.nvim', tag = '0.1.4',
-- or                              , branch = '0.1.x',
    dependencies = { 'nvim-lua/plenary.nvim' }
}
```
I prefer using the branch method.

Comment (roobert) [reply README.md:75]: Ok, I think the PR still needs updating to change this:

```
dependencies = {
    { 'nvim-telescope/telescope.nvim', branch = '0.1.x', dependencies = { 'nvim-lua/plenary.nvim' } }
```

to this?

```
  dependencies = {
    { 'nvim-telescope/telescope.nvim', branch = '0.1.x' }, { 'nvim-lua/plenary.nvim' } }
```

Unless I misunderstood and you're doing nested dependencies?

Comment (GotaLoveFiraCode) [reply README.md:75]: Yes, I was doing nested dependencies, though this isn't necessarily needed. I set edits to be allowed, so feel free to change this.

**Comment by roobert** [comment]
Thanks for your contribution!

**Pull request merged**